src/components/sessions/SessionList.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { gql, graphql } from 'react-apollo'
+
+import Session from './Session'
+
+const SessionList = ({ data }) =>
+  (<div>
+    {data.allSessions.map(session => <Session key={session.id} {...session} />)}
+  </div>)
+
+
+SessionList.propTypes = {
+  data: PropTypes.shape({
+    allSessions: PropTypes.arrayOf(
```

Comment:
There are ways to extract PropTypes from a GraphQL Schema so we don't have to repeat ourselves. Will have to look into it: https://github.com/apollographql/react-apollo/issues/302#issuecomment-262863047